### DISCUSSION TITLE
v6.0.0-alpha-1

### DISCUSSION THREAD

**Discussion by derkoe** [porscheinformatik/sonarqube-licensecheck#378] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/porscheinformatik/sonarqube-licensecheck/releases/tag/'>v6.0.0-alpha-1</a>.</em>

**Comment by Janpopan** [comment]
Hello @derkoe  there some 3rd party updates waiting. I think this major release is a good point to include them. What is your opinion?
Cheers
Jan
